.storybook/preview.js

```diff
@@ -17,7 +17,8 @@ export const parameters = {
     grid: { disable: true }
   },
   controls: {
-    hideNoControlsWarning: true
+    hideNoControlsWarning: true,
+    sort: 'alpha'
```

Comment:
No more `01-FileName.stories.mdx` 😉

Reply:
Oops, you're right @Francois-Esquire. I misstated the purpose of this sort, which is to ensure Controls are sorted alphabetically.

Reply:
I'd rather not use `storySort`, preferring rather to have the MDX files sort naturally in alphabetical order.